cli/crates/server/src/types.rs

```diff
@@ -23,18 +42,12 @@ pub enum ServerMessage {
     UdfMessage {
         udf_kind: UdfKind,
         udf_name: String,
-        level: UdfMessageLevel,
+        level: LogLevel,
         message: String,
     },
-    OperationStarted {
-        request_id: String,
-        name: Option<String>,
-    },
-    OperationCompleted {
+    OperationLogMessage {
         request_id: String,
-        name: Option<String>,
-        duration: std::time::Duration,
-        r#type: common::types::OperationType,
+        event_type: LogEventType,
```

Comment:
Maybe wrap `LogEvent` here? or is there a difference?

Reply:
One is the type for the bridge protocol. I guess there isn't a difference, but we have separate types for the other kinds of things happening (UDF invocation) to stress the separation of these protocols. I can take a look at merging these, but we should do it for all in that case.

Reply:
fair enough!